### DISCUSSION TITLE
Exist or not exist?

### DISCUSSION THREAD

**Discussion by leikoilja** [leikoilja/google-home-api#74] [Q&A]
Hey core team, @ArnyminerZ @KapJI, @DurgNomis-drol.
I wonder if we should archive/close this repo for now. Doesn't seem that any of us is particularly interested in investing to move all the API calls from `ha-google-home` to this repo, so perhaps there is no point of having it? :)

**Comment by DurgNomis-drol** [comment]
I actually have some code... 😃 I have used code from the integration and i started building it around using the eureka endpoint to fetch capabilities instead of the hacky way we do it now. Also did some research on only adding a specific home, which I don't think is possible, but i am not 100% sure about that yet.  Right now my code only contains getting info from the eureka endpoint to keep it simple, but should be easy to add the other ones too. Because of my dev setup, i realised that I first need to implement IP address list support in `glocaltokens` before i could continue developing, and i have gotten around to this yet. My code does also not have strict typing or anything yet, but it works.

**Reply by leikoilja** [reply]
That's amazing to hear, @DurgNomis-drol! Feel free to kick start the work on that with a PR, laying the good grounds for us to move the rest of API logic to this package instead 🚀

**Reply by DurgNomis-drol** [reply]
It should be pretty straight forward to add all other endpoints after this, as my main goal is to make it flexible and easy to extend. Will continue working on it when my PR in `glocaltokens` is done. 😃  May create a PR before it is done to get you guys inputs on how to best structure the library
